app/main.py

```diff
@@ -1,2 +1,72 @@
+from typing import Any
+
+INC_SIZE_TWICE = 2
+
+
 class Dictionary:
-    pass
+    size = 8
+
+    def __init__(self) -> None:
+        self.capacity = 0
+        self.hash_table = [None] * Dictionary.size
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        if isinstance(key, (list, dict, set)):
+            raise KeyError("key must be hashable")
```

Comment:
Code Efficiency: It's a good practice to provide a message when an exception raises. For example: `raise KeyError('key must be hashable')`

Reply:
I did the same